tests/test_menu.py

```diff
@@ -1,41 +1,49 @@
+import pytest
 from ttt.console_ui.menu import Menu
 from ttt.players.human_player import HumanPlayer
 from ttt.players.simple_computer_player import SimpleComputerPlayer
-from ttt.console_ui.console import Console
-from tests.mocks import MockConsole, MockGameRunner, MockConsoleIO
-from ttt.messages import welcome_message
+from tests.mocks import MockConsole, MockGameRunner
 
-def test_starting_a_game_requests_the_game_runner_to_run():
-    runner = MockGameRunner('console')
-    menu = Menu(MockConsole(inputs=['1', '', '1', '', 'n']), runner)
-    menu.start()
 
-    assert runner.run_call_count == 1
+class TestRunner:
+    def start_menu(self, inputs, game_runner):
+        menu = Menu(MockConsole(inputs), game_runner)
+        menu.start()
 
 
-def test_user_can_select_player_types():
-    runner = MockGameRunner('console')
-    menu = Menu(MockConsole(inputs=['1', '', '2', '', 'n']), runner)
-    menu.start()
+@pytest.fixture
+def test_runner():
+    return TestRunner()
 
-    assert isinstance(runner.player_1, HumanPlayer)
-    assert isinstance(runner.player_2, SimpleComputerPlayer)
 
+@pytest.fixture
+def game_runner():
+    return MockGameRunner('console')
 
-def test_if_player_one_selects_x_as_marker_player_two_default_marker_changes_to_o():
-    runner = MockGameRunner('console')
-    menu = Menu(MockConsole(inputs=['1', 'X', '1', '', 'n']), runner)
-    menu.start()
 
-    assert runner.player_2.get_marker() == 'O'
+def test_starting_a_game_requests_the_game_runner_to_run(test_runner, game_runner):
+    test_runner.start_menu(['1', '', '1', '', 'n'], game_runner)
```

Comment:
How would extracting the list of required inputs to start the game into a constant affect readability here?